libs/datamodel/core/src/dml/model.rs

```diff
@@ -189,7 +189,9 @@ impl Model {
         // first candidate: the singular id field
         {
             if let Some(x) = self.singular_id_fields().next() {
-                result.push(UniqueCriteria::new(vec![x]))
+                if !x.is_commented_out {
```

Comment:
Terminology doesn't check out for me. A model or field or whatever can be excluded, hidden, or any other form of term expressing absence, but "commenting out" is too specific to the representation of the matter at hand.